cli/metadata/actions/types/types.go

```diff
@@ -73,6 +73,7 @@ type ActionDef struct {
 	OutputType           string          `json:"output_type" yaml:"output_type,omitempty"`
 	ForwardClientHeaders bool            `json:"-" yaml:"forward_client_headers,omitempty"`
 	Headers              []yaml.MapSlice `json:"-" yaml:"headers,omitempty"`
+	Timeout              int             `json:"timeout" yaml:"timeout,omitempty""`
```

Comment:
@scriptonist do we need to export json?

Reply:
@arvi3411301 might not be necessary, since most of the operations on metadata deal with YAML. But thought to add it to confirm with other fields which seem to have `json` https://github.com/hasura/graphql-engine/blob/1f05a4e324bffe496edef1cd2d08f1644cd8f509/cli/metadata/actions/types/types.go#L68

Reply:
I think `timeout` is not handled by `cli-ext` plugin. We can ignore the json export.